frontend/src/components/common/FullPageLoading.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import styled from "styled-components";
```

Comment:
Somehow this file got reverted to about 2 weeks ago. I bring the changes back here